packages/test/test-end-to-end-tests/src/test/attachRegisterLocalApiTests.spec.ts

```diff
@@ -561,23 +561,18 @@ describeCompat(
 				const channel3 = await dataStore3.getSharedObject<ISharedMap>(mapId2);
 				assert.strictEqual(channel3.handle.isAttached, false, "Channel should be detached");
 
-				// dataStore2 POINTS TO dataStore3, channel3
-				// dataStore3 POINTS TO dataStore2, channel2
 				// channel2   POINTS TO dataStore3, channel3
 				// channel3   POINTS TO dataStore2, channel2
 				channel2.set("channel3handle", channel3.handle);
-				channel3.set("channel2handle", channel2.handle);
 				channel2.set("dataStore3", dataStore3.handle);
+				channel3.set("channel2handle", channel2.handle);
 				channel3.set("dataStore2", dataStore2.handle);
-				toFluidHandleInternal(dataStore2.handle).bind(
-					toFluidHandleInternal(dataStore3.handle),
-				);
-				toFluidHandleInternal(dataStore2.handle).bind(toFluidHandleInternal(channel3.handle));
-				toFluidHandleInternal(dataStore3.handle).bind(
-					toFluidHandleInternal(dataStore2.handle),
-				);
-				toFluidHandleInternal(dataStore3.handle).bind(toFluidHandleInternal(channel2.handle));
 
+				// Flow of attachment:
```

Comment:
These tests are strange as in they use internal constructs like `bind` and `attachGraph` in an end-to-end test which isn't what an application would do.
I would consider removing these tests or at least not doing this via internal functions. For instance, instead of calling `attachGraph()` here, dataStore2's handle can be stored in the defaultDataStore's DDS.

Reply:
Doesn't have to be in this PR of course. But these tests don't seem valuable to me.

Reply:
Agreed. I will create a work item to delete / update them.